### DISCUSSION TITLE
I started train model for cs2 hs mod 8k images

### DISCUSSION THREAD

**Discussion by Alex-test-mod** [Babyhamsta/Aimmy#1391] [Show and tell]
![labels_correlogram](https://github.com/Babyhamsta/Aimmy/assets/34565256/ef4cb0a8-4ae0-491f-a807-fabcd1da9db4)
![labels](https://github.com/Babyhamsta/Aimmy/assets/34565256/55fab384-a502-436c-b593-ff1a4ee2dd7b)

**Comment by AL99117** [comment]
Can you share your CS2 noox with me?

**Reply by Alex-test-mod** [reply]
https://github.com/Babyhamsta/Aimmy/blob/master/models/cstest5000.onnx

**Reply by Alex-test-mod** [reply]
CT and T model

**Comment by AL99117** [comment]
thanks
